build/isc-bind9/bind.p5m

```diff
@@ -0,0 +1,13 @@
+set name=pkg.fmri value=ooce/network/bind-common@1.0.0-@PVER@
+set name=variant.arch value=i386
+set name=variant.opensolaris.zone value=global value=nonglobal
+set name=description value="Common BIND package"
+set name=pkg.description value="Common BIND package"
+set name=pkg.summary value="Common BIND package"
+
+dir group=bin mode=0755 owner=named path=var/opt/ooce/bind
+dir group=bin mode=0755 owner=named path=var/log/opt/ooce/bind
+
+group groupname=named gid=93
+user ftpuser=false username=named uid=93 group=named gcos-field="BIND User" \
```

Comment:
Can you use uid/gid 53?
Also please update `doc/idlist` to record these.

Reply:
this is done